namer/namer.cc

```diff
@@ -1215,6 +1215,11 @@ class SymbolDefiner {
 
                 auto singletonClass = symbol.data(ctx)->singletonClass(ctx); // force singleton class into existence
                 singletonClass.data(ctx)->addLoc(ctx, ctx.locAt(klass.declLoc));
+                for (const auto &typeMember : singletonClass.data(ctx)->typeMembers()) {
+                    if (typeMember.data(ctx)->name == core::Names::Constants::AttachedClass()) {
+                        typeMember.data(ctx)->addLoc(ctx, ctx.locAt(klass.declLoc));
+                    }
+                }
```

Comment:
```suggestion
                auto attachedClassTM =
                    singletonClass.data(ctx)->findMember(ctx, core::Names::Constants::AttachedClass());
                if (attachedClassTM.exists() && attachedClassTM.isTypeMember()) {
                    attachedClassTM.asTypeMemberRef().data(ctx)->addLoc(ctx, ctx.locAt(klass.declLoc));
                }
```

No need to loop when we can find.